line_detection_node:
  camera_info_topic_name: "camera/color/camera_info"
  depth_topic_name: "camera/aligned_depth_to_color/image_raw"
  rgb_topic_name: "camera/color/image_raw"
  white_region_threshold: 180

velocity_estimation:
  camera_info_topic_name: "camera/color/camera_info"
  visual_measure_topic_name: "visual_measure"
  vehicle_imu_topic_name: "mavros/imu/data"
  depth_topic_name: "camera/aligned_depth_to_color/image_raw"